framework/h5py_interface_creator.py

```diff
@@ -134,13 +134,14 @@ def __init__(self,name, databaseDir, messageHandler, filename, exist, variables
       # self.h5FileW is the HDF5 object. Open the database in "write only" mode
       self.h5FileW = self.openDatabaseW(self.filenameAndPath,'w')
       # Add the root as first group
-      self.allGroupPaths.append("/")
+      self.allGroupPaths.append(b"/")
```

Comment:
Hm, this will probably cause other places to need b"..."

Reply:
I think I fixed all of them